### PR TITLE
add random color

### PR DESCRIPTION
**Pull request by GusCaplan**

### CODE REVIEW

File: src/client/ClientDataResolver.js [hunk 1]
```diff
@@ -360,6 +361,7 @@ class ClientDataResolver {
    */
   static resolveColor(color) {
     if (typeof color === 'string') {
+      if (color === 'RANDOM') return Math.floor(Math.random() * 16777214) + 1;
```
Comment (meew0) [inline src/client/ClientDataResolver.js]: My comment from the old PR still applies:

> Won't the minimum colour be 1 in this case? 0 is also a valid colour (black).

Comment (GusCaplan) [inline src/client/ClientDataResolver.js]: i had a reason that i made it 1, but i don't remember what it was. if i can't remember it in four hours from this posting i will make it 0

Comment (Gawdl3y) [inline src/client/ClientDataResolver.js]: "Uhh, technically black isn't a colour - it's the lack of colour, you stupid dimwit."
- Gus

Comment (GusCaplan) [inline src/client/ClientDataResolver.js]: Let it be known that I never said that 👀

**Pull request merged**